src/transport/raw/BLE.cpp

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
Any chance to do a `git mv`  to see a diff between this file and the original `src/transport/BLE.cpp` ?

Reply:
My git actually think raw/BLE.h is a rename, however, seems github does not consider it as a rename since its similarity is too low.

Reply:
I split it into three commits.
1. The first commit is the logic chanes, i.e. remove rendezvous events from BLE, add "send after connect", etc.
2. The second commit just moved it to raw/
3. The third commit makes some naming changes, and makes the code clearer.